src/builder/create_application_command_permission.rs

```diff
@@ -71,7 +73,7 @@ impl CreateApplicationCommandPermissions {
         self
     }
 
-    /// Creates a new permission for the interaction.
+    /// Creates permissions(s) for the application command.
```

Comment:
```suggestion
    /// Creates permissions for the application command.
```
I've never seen someone use singular permission when setting permissions in computer terminology even if it is a singular permission.

Reply:
I wanted to make a distinction, even if it is not a very common terminology. I'll change that